### PR TITLE
feat(16988): shared admin page

### PR DESCRIPTION
**Pull request by nnorbert**
shared admin page

Screenshot 2024-07-26 at 14 35 21

### CODE REVIEW

File: apps/home/app/styles/globals.scss [hunk 1]
```diff
@@ -1,27 +1,80 @@
...
 @mixin phone {
-  @media (max-width: ($breakpoint-phone)) {
+  @media (max-width: ($breakpoint-md)) {
     @content;
   }
 }
 
 @mixin tablet {
-  @media (min-width: ($breakpoint-phone + 1)) and (max-width: ($breakpoint-tablet)) {
+  @media (min-width: ($breakpoint-md + 1)) and (max-width: ($breakpoint-lg)) {
```
Comment (peschina) [inline apps/home/app/styles/globals.scss:65]:
```suggestion
  @media (min-width: ($breakpoint-md)) and (max-width: ($breakpoint-lg)) {
```

File: apps/home/app/[locale]/components/Dashboard/dashboard.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 20px;
+  text-decoration: none;
+  width: 100px;
+}
+
+@media (max-width: 767px) {
```
Comment (peschina) [inline apps/home/app/[locale]/components/Dashboard/dashboard.css:31]:
```suggestion
@media (max-width: $breakpoint-md) {
```

With `@import "/app/styles/globals.scss";` on top

File: apps/home/app/[locale]/components/HamburgerMenu/HamburgerMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+type MenuProps = {
+  userName: string;
+  publicServant: boolean;
+  handleClick: () => void;
+  languageSwitch: React.ReactNode;
+  logout: React.ReactNode;
+};
+
+const options: Array<{
```
Comment (peschina) [inline apps/home/app/[locale]/components/HamburgerMenu/HamburgerMenu.tsx:19]: I guess we can remove this const

File: apps/home/app/[locale]/components/HamburgerMenu/HamburgerMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                  <Icon
+                    icon={option.icon as IconProps["icon"]}
+                    className="govie-button__icon-left"
+                    color={ds.colours.ogcio.darkGreen}
+                  />
+                  {t(option.label)}
+                </Link>
+              </li>
+            );
+          })}
```
Comment (peschina) [inline apps/home/app/[locale]/components/HamburgerMenu/HamburgerMenu.tsx:94]: We can remove this as well, right?

File: apps/home/app/[locale]/components/Header/Header.tsx [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+import { AuthenticationFactory } from "../../../../libraries/authentication-factory";
+import { getTranslations } from "next-intl/server";
+
+type HeaderProps = {
+  locale: string;
+};
+
+const showLogin = () => {
+  return String(process.env.ALLOW_LOGIN) === "true";
+};
```
Comment (peschina) [inline apps/home/app/[locale]/components/Header/Header.tsx:18]: Would this work as well?
`const showLogin = String(process.env.ALLOW_LOGIN) === "true";`

File: apps/home/app/styles/globals.scss [hunk 6]
```diff
@@ -1,27 +1,80 @@
...
     @content;
   }
 }
 
 @mixin desktop {
-  @media (min-width: ($breakpoint-tablet + 1)) and (max-width: ($breakpoint-desktop)) {
+  @media (min-width: ($breakpoint-lg + 1)) and (max-width: ($breakpoint-xl)) {
```
Comment (peschina) [inline apps/home/app/styles/globals.scss:71]:
```suggestion
  @media (min-width: ($breakpoint-lg)) and (max-width: ($breakpoint-xl)) {
```

File: apps/home/app/styles/globals.scss [hunk 7]
```diff
@@ -1,27 +1,80 @@
...
     @content;
   }
 }
 
 @mixin wide-desktop {
-  @media (min-width: ($breakpoint-desktop + 1)) {
+  @media (min-width: ($breakpoint-xl + 1)) {
```
Comment (peschina) [inline apps/home/app/styles/globals.scss:77]:
```suggestion
  @media (min-width: ($breakpoint-xl)) {
```

**Review by peschina — COMMENTED** [review]
A few minor comments, but looks good! If you agree merge the build time environment variables addiction here first

File: apps/home/.env.sample [hunk 8]
```diff
@@ -5,3 +5,15 @@ ENVIRONMENT=DEV
...
+LOGTO_HOME_APP_ID=home_app_id
+LOGTO_HOME_APP_SECRET=home_app_local_secret
+LOGTO_COOKIE_SECRET=home_app_local_secret
+
+NEXT_PUBLIC_HOME_SERVICE_ENTRY_POINT = http://localhost:3004
+NEXT_PUBLIC_PAYMENTS_SERVICE_ENTRY_POINT = http://localhost:3001
+NEXT_PUBLIC_MESSAGING_SERVICE_ENTRY_POINT = http://localhost:3002
+NEXT_PUBLIC_LIFE_EVENTS_SERVICE_ENTRY_POINT = http://localhost:3000
+NEXT_PUBLIC_FORMS_SERVICE_ENTRY_POINT = '#'
+NEXT_PUBLIC_DESIGN_SYSTEM_SERVICE_ENTRY_POINT = '#'
```
[the remark this answers is not in the document]
Comment (nnorbert) [reply apps/home/.env.sample]: Done!

File: apps/home/app/[locale]/components/HamburgerMenu/HamburgerMenu.tsx (hunk 3 above)
Comment (nnorbert) [reply apps/home/app/[locale]/components/HamburgerMenu/HamburgerMenu.tsx:19]: Done

File: apps/home/app/[locale]/components/Header/Header.tsx (hunk 5 above)
Comment (nnorbert) [reply apps/home/app/[locale]/components/Header/Header.tsx:18]: Done

**Review by peschina — APPROVED** [review]
LGTM

File: apps/home/app/[locale]/components/Dashboard/index.tsx [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+  },
+  {
+    url: process.env.NEXT_PUBLIC_LIFE_EVENTS_SERVICE_ENTRY_POINT ?? "#",
+    labelKey: "lifeEvents",
+    icon: "events",
+    role: "Life Events Public Servant",
+  },
+];
+
+const translations = {
```
Comment (alfonsograziano) [inline apps/home/app/[locale]/components/Dashboard/index.tsx:50]: non-blocking: is there any reason for not using the translation files instead like we're doing in the other applications?

File: apps/messaging/app/utils/logto-config.ts [hunk 10]
```diff
@@ -26,7 +26,7 @@ const publicServantScopes = [
   "messaging:citizen:*",
   "messaging:event:read",
 ];
-const publicServantExpectedRole = "Messaging Public Servant";
+const publicServantExpectedRoles = ["Messaging Public Servant"];
```
Comment (alfonsograziano) [inline apps/messaging/app/utils/logto-config.ts:29]: @SamSalvatico FYI

**Review by alfonsograziano — DISMISSED** [review]
LGTM

File: apps/home/app/[locale]/components/Dashboard/index.tsx (hunk 9 above)
Comment (nnorbert) [reply apps/home/app/[locale]/components/Dashboard/index.tsx:50]: Oh... I just copied this part and I forgot to move them into the translation file. I will update it soon.

**Review by alfonsograziano — APPROVED** [review]

**Pull request merged**